static/css/restyle.less

```diff
@@ -63,6 +63,25 @@ h1 {
   font-size: 2.25rem;
 }
 
+// These are all defined in AMO's original CSS and seem to need overriding :'(
+a,
```

Comment:
`main.css` seriously had a selector that basically looked like this. I figured it was better to replace it than to use `!important`.